schema/schemahandler.go

```diff
@@ -31,6 +32,30 @@ func NewPathMap(path string) *PathMapType {
 	return pathMap
 }
 
+// Printout the structure of the schema path.
+// Ex.
+// Parquet_go_root
+//
+//	Sex: Parquet_go_rootSex
+//	Name: Parquet_go_rootName
+//	Age: Parquet_go_rootAge
+//	Classes: Parquet_go_rootClasses
+//	  List: Parquet_go_rootClassesList
+//	    Element: Parquet_go_rootClassesListElement
+//	      Name: Parquet_go_rootClassesListElementName
+//	      Number: Parquet_go_rootClassesListElementNumber
+//	      Score: Parquet_go_rootClassesListElementScore
```

Comment:
I **think** you're missing some spaces here. For instance, I assume you wanted:

```
Number: Parquet_go_root Classes List Element Number
```

As opposed to:

```
Number: Parquet_go_rootClassesListElementNumber
```

So you probably need to make some minor adjustments in the `output` method's implementation?

Reply:
I didn't update its underlying path string. There is no space there, but it has a special character \x01 which is probably skipped in the print string. I don't think it's worth to split the path given the purpose only for us to understand the structure.

Reply:
In case you don't think this would be hard to parse/inspect later down the line without the space, which would make it easier to quickly see what's going on (especially IF we'll be dealing with more complicated and nested data structures), then sounds good to me 😄